views/layouts/default_email.hbs

```diff
@@ -54,6 +54,11 @@
         border-width: 0px;
       }
 
+      p, 
+      a {
+        font-family: sans-serif;
+      }
+
```

Comment:
Heh, I almost said "no inline CSS!" and then I remember ... email. 😢

Reply:
HTMLing like it's 1997.